docs/modelfile.md

```diff
@@ -132,7 +133,7 @@ PARAMETER <parameter> <parametervalue>
 
 ### TEMPLATE
 
-`TEMPLATE` of the full prompt template to be passed into the model. It may include (optionally) a system prompt and a user's prompt. This is used to create a full custom prompt, and syntax may be model specific.
+`TEMPLATE` of the full prompt template to be passed into the model. It may include (optionally) a system prompt and a user's prompt. This is used to create a full custom prompt, and syntax may be model specific. You can usually find the template for a given model in the readme for that model.
```

Comment:
This should link to the modelfile doc section on templates

> You can usually find the template for a given model in the readme for that model.

Let's make this more specific (what is "the readme"?) I think examples can be in the modelfile doc

Reply:
this is the modelfile doc. Not sure what you are asking.